Dockerfile

```diff
@@ -44,12 +44,6 @@ RUN chmod a+x /usr/local/bin/check-synced.sh
 
 WORKDIR /data
 
-# Electrum RPC
-EXPOSE 50001
-
-# Prometheus monitoring
-EXPOSE 4224
-
```

Comment:
These lines were never needed, actually, as StartOS doesn't obey the Dockerfile's EXPOSE directives.  That's handled in manifest.yaml's interfaces section:
https://github.com/Start9Labs/electrs-startos/blob/update/v0.10.9/manifest.yaml#L60-L74
Currently, we open 50001 only.